src/mono/mono/mini/interp/interp.c

```diff
@@ -2169,19 +2169,73 @@ dump_args (InterpFrame *inv)
 #define CHECK_MUL_OVERFLOW_NAT_UN(a,b) CHECK_MUL_OVERFLOW64_UN(a,b)
 #endif
 
+static MONO_NEVER_INLINE void
+interp_runtime_invoke_cctor (MonoMethod *method, MonoObject **exc, MonoError *error)
+{
+	ThreadContext *context = get_context ();
+	stackval *sp = (stackval*)context->stack_pointer;
+	InterpMethod *imethod = mono_interp_get_imethod (method);
+	g_assert (!imethod->transformed);
```

Comment:
why?

Reply:
cctors shouldn't run twice, and I put that there to verify that I understood how all of this code flows